packages/github/src/client/GitHub.mjs

```diff
@@ -1,21 +1,21 @@
 import { Logger } from "@aws-lambda-powertools/logger";
 import { Octokit } from "@octokit/rest";
+import { z } from "zod";
 import {
 	MAX_ISSUES_LIMIT,
 	MAX_ISSUES_PER_PAGE,
 	MAX_PULL_REQUESTS_LIMIT,
 	MAX_PULL_REQUESTS_PER_PAGE,
 	SERVICE_NAME,
-} from "github/src/constants.mjs";
-import { issueSchema, issueSearchSchema } from "github/src/schemas/issues.mjs";
-import { pullRequestSchema } from "github/src/schemas/pull_requests.js";
-import { z } from "zod";
+} from "../constants.mjs";
```

Comment:
for my own understanding... we use relative path otherwise it breaks subpath exports etc?